qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -81,10 +86,21 @@ class QuteCodeActions {
 
 	private static final String UNDEFINED_NAMESPACE_SEVERITY_SETTING = "qute.validation.undefinedNamespace.severity";
 
+	private static final String APPEND_TO_TEMPLATE_EXTENSIONS = "Create template extension \"%s()\" in detected template extensions class";
+	
+	private static final String CREATE_TEMPLATE_EXTENSIONS = "Create template extension \"%s()\" in a new template extensions class";
+
+	private static final String CREATE_GETTER = "Create getter \"get%s()\" in %s";

```

Comment:
Use ` instead of `"` and ends with . your message to be consistent with other code action title:

![image](https://user-images.githubusercontent.com/1932211/181309705-d49a378a-b33f-4847-976f-6388bc54d7ce.png)

Reply:
It is not fixed, no?